### DISCUSSION TITLE
Including PASTA Methodology and Auto-Generations Threat Capabilities.

### DISCUSSION THREAD

**Discussion by 7Shadow7** [OWASP/threat-dragon#681] [Ideas] [answered]
Adding PASTA Methodology in Threat Dragon is plus point and Auto Threat Generation capabilities should be very effective features for Threat Dragon. Showcasing Risk related with the Threats would make this application more effective for Threat Modelling

**Comment by jgadsden** [comment, marked as answer]
I agree @7Shadow7 that we could consider PASTA methodology and should raise an issue for this
I have raised an issue #792 to reinstate automated generation of threats by element
